nodejs/index.js

```diff
@@ -3,18 +3,17 @@
 // Imports
 const express = require('express');
 const bodyParser = require('body-parser');
-const Smooch = require('smooch-core');
+const SunshineConversationsApi = require('./dist/smooch-core-v2');
```

Comment:
Eventually will need to be updated to sunshine-conversations-client ?
```suggestion
const SunshineConversationsApi = require('./dist/sunshine-conversations-client');
```

Reply:
Will update once the sunco library is in npm